stripe/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsActivity.kt

```diff
@@ -143,7 +143,7 @@ internal class PaymentOptionsActivity : BaseSheetActivity<PaymentOptionResult>()
                     // where we also jump to a new unsaved card. However this move require
                     // the transition target to specify when to and when not to add things to the
                     // backstack.
-                    if (starterArgs.paymentMethods.isEmpty()) {
+                    if (starterArgs.paymentMethods.isEmpty() && viewModel.isGooglePayReady.value?.not() == true) {
```

Comment:
To confirm the behavior, when there is no customer or the customer doesn't have payment methods AND Google Pay is ready, we show the carousel?

Reply:
yes, if google pay is enabled and there is no customer, or no payment methods we want to show the carousel